### PR TITLE
Add council composition to the readme

### LINKED ISSUE #26: Adding a list of Coucil members to team compass
Hi Jupyter Widgets Council. Recently the Jupyter Executive Council needed to make a list of all the Subproject Council members. In the process of compiling that list we discovered that Jupyter Widgets didn’t have a public list of Council members that we could find. The EC asks that you create a list of your Council members in your team compass. Thanks!

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Fixes #26

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
Ping @ibdafna @martinRenou @maartenbreddels

File: README.md [hunk 1]
```diff
@@ -7,3 +7,26 @@ A repository for team discussion, syncing, and meeting notes.
...
+David Brochart
+Sylvain Corlay
+Matt Craig
+Itay Dafna
+Vidar Tonaas Fauske
+Jason Grout
+Paul Ivanov
+Mariana Meireles
+Jeremy Tuloup
+Martin Renou
```
Comment (jtpio) [inline README.md]: Maybe using a list so the names render better?

image

```suggestion
- David Brochart
- Sylvain Corlay
- Matt Craig
- Itay Dafna
- Vidar Tonaas Fauske
- Jason Grout
- Paul Ivanov
- Mariana Meireles
- Jeremy Tuloup
- Martin Renou
```

Comment (maartenbreddels) [reply README.md]: I can't do a suggestion on a suggestion, can you add my name? :)

**Review by martinRenou — APPROVED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (SylvainCorlay) [reply README.md]: Apologies, I missed it when I re-ordered the list (I did ping you though 😄 ). It is fixed now.

**Pull request merged**